server/mailet/integration-testing/src/test/java/org/apache/james/mailets/configuration/ProcessorConfiguration.java

```diff
@@ -61,7 +63,7 @@ public Builder addMailet(MailetConfiguration mailetConfiguration) {
 
         public ProcessorConfiguration build() {
             Preconditions.checkState(!Strings.isNullOrEmpty(state), "'state' is mandatory");
-            return new ProcessorConfiguration(state, enableJmx, mailets.build());
+            return new ProcessorConfiguration(state, enableJmx.orElse(false), mailets.build());
```

Comment:
ok I know you are a big fan of optionals, but having just boolean enableJmx=false at the beginning would have done the trick, no? BTW it's pretty well known that booleans default to false, no?

Reply:
```
it's pretty well known that booleans default to false
``` 

I did not.

Using an optional for default value in builder is **team standard** thus not having it seems awkward to me. Let's have an homogeneus code base